src/current/_includes/v23.1/sidebar-data/reference.json

```diff
@@ -1784,35 +665,6 @@
             ]
           }
         ]
-      },
-      {
-        "title": "CockroachDB Feature Availability",
-        "urls": [
-          "/${VERSION}/cockroachdb-feature-availability.html"
-        ]
-      },
-      {
-        "title": "Unsupported Features in CockroachDB Serverless",
-        "urls": [
-          "/cockroachcloud/serverless-unsupported-features.html"
-        ]
-      },
-      {
-        "title": "Third-Party Support",
-        "items": [
-          {
-            "title": "Tools Supported by Cockroach Labs",
-            "urls": [
-              "/${VERSION}/third-party-database-tools.html"
-            ]
-          },
-          {
-            "title": "Tools Supported by the Community",
-            "urls": [
-              "/${VERSION}/community-tooling.html"
-            ]
-          }
-        ]
       }
     ]
   }
```

Comment:
Just confirming that these items still under Reference will be removed in a future phase:

<img width="210" alt="image" src="https://github.com/cockroachdb/docs/assets/8740013/391f1784-fc2a-407a-a160-17a0c4c0ed15">

Reply:
Yes, these will be addressed in the next 2 phases